src/bin/casr-python.rs

```diff
@@ -0,0 +1,296 @@
+            if let Some(cap) = re.captures(&report.python_report[1]) {
+                report.execution_class.severity = Cow::Borrowed("UNDEFINED");
+                report.execution_class.short_description =
+                    Cow::Borrowed(cap.get(1).unwrap().as_str());
+                report.execution_class.description = Cow::Borrowed(cap.get(2).unwrap().as_str());
+                report.execution_class.explanation = Cow::Borrowed("");
+            }
+
+            // Get stack trace from python report.
+            let first = report
+                .python_report
+                .iter()
+                .position(|line| line.starts_with("Traceback "));
+            if first.is_none() {
+                bail!("Couldn't find traceback in python report");
+            }
+
+            // Stack trace is splitted by empty line.
+            let first = first.unwrap();
+            let last = report
+                .python_report
+                .iter()
+                .skip(first)
+                .rposition(|s| !s.is_empty());
+            if last.is_none() {
+                bail!("Couldn't find traceback end in python report");
+            }
+            let last = last.unwrap();
+
+            let re =
+                Regex::new(r#"(File "[\w\-<>~./]+", line [\d]+, in [\w<>]+|\[Previous line repeated (\d+) more times\])"#).unwrap();
+            report.stacktrace = report.python_report[first..first + last]
+                .iter()
+                .rev()
+                .map(|s| s.trim().to_string())
+                .filter(|s| re.is_match(s))
+                .collect::<Vec<String>>();
+        } else {
+            bail!("Need to use casr-san with LD_PRELOAD")
+        }
+    } else if let Some(report_start) = python_stderr_list
+        .iter()
+        .position(|line| line.contains("Traceback "))
+    {
+        // Set python report in casr report.
+        let report_end = python_stderr_list
+            .iter()
+            .rposition(|s| !s.is_empty())
+            .unwrap()
+            + 1;
+        report.python_report = Vec::from(&python_stderr_list[report_start..report_end]);
+
+        // Get stack trace from python report.
+        let first = report
+            .python_report
+            .iter()
+            .position(|line| line.starts_with("Traceback "));
+        if first.is_none() {
+            bail!("Couldn't find traceback in python report");
+        }
+
+        // Stack trace is splitted by empty line.
+        let first = first.unwrap();
+        let last = report
+            .python_report
+            .iter()
+            .skip(first)
+            .rposition(|s| !s.is_empty());
+        if last.is_none() {
+            bail!("Couldn't find traceback end in python report");
+        }
+        let last = last.unwrap();
+
+        let re =
+            Regex::new(r#"(File "[\w\-<>~./]+", line [\d]+, in [\w<>]+|\[Previous line repeated (\d+) more times\])"#).unwrap();
```

Comment:
this regexp is common with the one in python.rs, maybe we can make a constant variable for it

Reply:
No, they are different